README.md

```diff
@@ -1935,6 +1935,19 @@ $ brew unlink trivy && brew uninstall trivy
 $ brew install aquasecurity/trivy/trivy
 ```
 
+### Error: FATAL	unable to initialize the cache: failed to create cache dir: mkdir /root/.cache: permission denied
+
+```
+$ docker run --rm -v $PWD:/root/.cache/  aquasec/trivy:0.9.2 --cache-dir /root/.cache image centos:7
+2020-07-29T15:02:54.435Z	FATAL	unable to initialize the cache: failed to create cache dir: mkdir /root/.cache: permission denied
+```
+
+Try:
+
+```
+$ docker run --rm -v $PWD:/tmp/.cache/  aquasec/trivy:0.9.2 --cache-dir /tmp/.cache image centos:7
```

Comment:
For the version numbers maybe it's better to show people to use `latest` than to hardcode with a particular version. This could be useful in case people just copy-pasta commands to run them.

Reply:
@simar7  Done